src/emqtt.erl

```diff
@@ -1036,6 +1036,20 @@ terminate(Reason, _StateName, State = #state{conn_mod = ConnMod, socket = Socket
         _ -> ConnMod:close(Socket)
     end.
 
+code_change({down, OldVsn}, State, Data, _Extra) when OldVsn == "1.2.3",
```

Comment:
As far as I know, we still have an older version of 1.2.3.1. Is this version also allowed to be upgraded to 1.2.4?